platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -673,7 +757,83 @@ def _read_eeprom_specific_bytes(self, offset, num_bytes):
         except subprocess.CalledProcessError as e:
             return None
 
-        return eeprom_raw
```

Comment:
@ayurkiv-nvda why do we need this?

Reply:
partial alignment with sfp refactoring  https://github.com/Azure/sonic-buildimage/pull/10317

Reply:
@ayurkiv-nvda partial alignment adds a mess: please consider either full alignment or simply avoid that. Once the dependent PR is merged you will revisit the current changes